pandas/core/generic.py

```diff
@@ -1865,17 +1865,22 @@ def to_sql(self, name, con, schema=None, if_exists='fail', index=True,
         """
         Write records stored in a DataFrame to a SQL database.
 
+        This function inserts all rows of the dataframe into the given
+         table and recreates it if if_exists='replace'. Databases supported by
+         SQLAlchemy or DBAPI2 are also supported.
+
         Parameters
         ----------
         name : string
-            Name of SQL table
+            Name of SQL table.
         con : SQLAlchemy engine or DBAPI2 connection (legacy mode)
             Using SQLAlchemy makes it possible to use any DB supported by that
             library. If a DBAPI2 object, only sqlite3 is supported.
         schema : string, default None
             Specify the schema (if database flavor supports this). If None, use
             default schema.
         if_exists : {'fail', 'replace', 'append'}, default 'fail'
+            Accepted values:
```

Comment:
FWIW, I find it clearer without the line here, though I know the script complains when it isn't present :)  cc @datapythonista